python_modules/dagster/dagster/_core/definitions/asset_check_spec.py

```diff
@@ -5,6 +5,12 @@
 from dagster._core.definitions.events import AssetKey, CoercibleToAssetKey
 
 
+@experimental
+class AssetCheckHandle(NamedTuple):
+    name: PublicAttr[str]
+    asset_key: PublicAttr[AssetKey]
```

Comment:
probably worth a little doc block (explaining that check names are unique or per-asset basis, for example)

Reply:
👍